code/modules/surgery/burn_dressing.dm

```diff
@@ -0,0 +1,175 @@
+	..()
+	display_results(user, target, "<span class='notice'>You carve away some of the healthy flesh from [target]'s [parse_zone(target_zone)].</span>",
+		"<span class='notice'>[user] carves away some of the healthy flesh from [target]'s [parse_zone(target_zone)] with [tool]!</span>",
+		"<span class='notice'>[user] carves away some of the healthy flesh from  [target]'s [parse_zone(target_zone)]!</span>")
+	surgery.operated_bodypart.receive_damage(brute=rand(4,11), sharpness=TRUE)
+
+///// Regenerate, remove flesh_damage
+/datum/surgery_step/regenerate_flesh
+	name = "regenerate flesh (mesh)"
+	implements = list(/obj/item/stack/medical/mesh = 100)
+	time = 40
+	repeatable = TRUE
+	experience_given = MEDICAL_SKILL_MEDIUM
+
+/datum/surgery_step/regenerate_flesh/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/stack/medical/tool, datum/surgery/surgery)
+	if(surgery.operated_wound)
+		if(surgery.operated_wound.flesh_damage <= 0)
+			to_chat(user, "<span class='notice'>[target]'s [parse_zone(user.zone_selected)] is as healthy flesh-wise as you'll get!</span>")
+			return
+		display_results(user, target, "<span class='notice'>You begin to carefully apply [tool] to [target]'s [parse_zone(user.zone_selected)]...</span>",
+			"<span class='notice'>[user] begins to carefully apply [tool] to [target]'s [parse_zone(user.zone_selected)].</span>",
+			"<span class='notice'>[user] begins to carefully apply [tool] to [target]'s [parse_zone(user.zone_selected)].</span>")
+	else
+		user.visible_message("<span class='notice'>[user] looks for [target]'s [parse_zone(user.zone_selected)].</span>", "<span class='notice'>You look for [target]'s [parse_zone(user.zone_selected)]...</span>")
+
+/datum/surgery_step/regenerate_flesh/success(mob/user, mob/living/carbon/target, target_zone, obj/item/stack/medical/tool, datum/surgery/surgery, default_display_results = FALSE)
+	if(surgery.operated_wound)
+		display_results(user, target, "<span class='notice'>You successfully apply some of [tool] to [target]'s [parse_zone(target_zone)].</span>",
+			"<span class='notice'>[user] successfully applies some of [tool] to [target]'s [parse_zone(target_zone)] with [tool]!</span>",
+			"<span class='notice'>[user] successfully applies some of [tool] to [target]'s [parse_zone(target_zone)]!</span>")
+		log_combat(user, target, "regenerated flesh in", addition="INTENT: [uppertext(user.a_intent)]")
+		surgery.operated_wound.sanitization += tool.sanitization * 1.5
+		surgery.operated_wound.flesh_healing += tool.regenerate_flesh * 1.5
+		if(sanitization >= infestation && flesh_healing > flesh_damage)
+			repeatable = FALSE
+	else
+		to_chat(user, "<span class='warning'>[target] has no ruined flesh there!</span>")
+	return ..()
+
+*/
+///// Dressing burns
+/datum/surgery_step/dress
+	name = "dress burns (bandage)"
+	implements = list(/obj/item/stack/medical/gauze = 100, /obj/item/stack/sticky_tape/surgical = 100)
```

Comment:
why are no ghetto equivalents available for this step?

Reply:
I mean, I guess the "ghetto equivalent" for this step is using a cautery and drapes to exit the surgery without actually doing this step, but still...

Reply:
oh, wait, can you use improvised gauze for this step?